config/config.go

```diff
@@ -425,6 +434,9 @@ func initConsensusProtocols() {
 	vFuture.LogicSigMaxSize = 1000
 	vFuture.LogicSigMaxCost = 20000
 	vFuture.MaxAssetsPerAccount = 1000
+	vFuture.MaxAssetNameBytes = 32
+	vFuture.MaxAssetUnitNameBytes = 8
+	vFuture.MaxAssetURLBytes = 32
```

Comment:
We can no longer make these changes. However, since these are all new parameters, you could add them to v7 as the default ?